Remotion/Web/Development.WebTesting.IntegrationTests.Infrastructure/CustomWebTestConfigurationFactory.cs

```diff
@@ -96,6 +98,26 @@ protected override IBrowserConfiguration CreateCustomBrowserConfiguration (WebTe
       return base.CreateCustomBrowserConfiguration (configSettings);
     }
 
+    private string GetVersionedBrowserFolderName (string browserName, string versionArchivePath, string latestTestedMajorBrowserVersion)
+    {
+      return Directory.GetFiles (versionArchivePath)
+          .Select (Path.GetFileNameWithoutExtension)
+          .Where (x => x.StartsWith (browserName))
+          .ToDictionary (fileName => fileName, fileName => CreateVersionFromBrowserZipName (fileName, browserName))
+          .OrderByDescending (kvp => kvp.Value)
+          .First (kvp => kvp.Value.Major == int.Parse (latestTestedMajorBrowserVersion))
+          .Key;
+    }
+
+    private Version CreateVersionFromBrowserZipName (string fileName, string browserName)
+    {
+      var regex = new Regex (@$"^{browserName}_v(?<major>\d*)(?<additionalVersionInfo>(\.\d+)*)");
+      var match = regex.Match (fileName);
+      return new Version ($"{match.Groups["major"].Value}{NormalizeAdditionalVersionInfo (match.Groups["additionalVersionInfo"].Value)}");

```

Comment:
How about
`return new Version ($"{match.Groups["major"].Value}{StringUtility.EmptyToNull (match.Groups["additionalVersionInfo"].Value) ?? ".0"}");` ?

Reply:
Is putting the latter part in a variable okay? It's a little too much for a single line, don't you think? But using `StringUtility` is cool